ProcessOptimizer/space/space.py

```diff
@@ -420,14 +462,50 @@ def lhs_arange(self, n):
         # and picking the middle of each piece. Samples are spaced 1/n apart
         # inside the interval with a buffer of half a step size to the extremes
         samples = (np.arange(n)+0.5)/n
-        # Transform the samples to the range used for this dimension and then 
-        # round them back to integers. If your space is less than n wide, some 
+        # Transform the samples to the range used for this dimension and then
+        # round them back to integers. If your space is less than n wide, some
         # of your samples will be rounded to the same number
         samples = np.round(samples*(self.high - self.low) + self.low)
-        
+
         # Convert samples to a list of integers
         return samples.astype(int)
 
+    def evenly_sample(self, num_points, border_width=None):
+        """Return `n_points` evenly spaced points from a Dimension.
+
+        Parameters
+        ----------
+        * `num_points` [int]
+            The number of points to sample from `dim`.
+
+        * `border_width` [int]
+            How wide to make the borders. If None, no borders are returned.

```

Comment:
Again, is it intentional that this input is an int? Is the border simply in the relevant dimension scale?